docs/agent/changes.md

```diff
@@ -326,6 +326,8 @@ are ported, excepted the following deprecations:
     Kubernetes and Openshift (default to true). This will avoid removing
     them from the exclude list by error
 
+The format of the `DD_TAGS`  global option has changed. In v5 it is comma-separated; in v6 it is space-separated, i.e. `simple-tag-0 tag-key-1:tag-value-1`.
```

Comment:
I think we should remove this line. It's under the `docker check` section whereas it actually applies to the containerized Agent, which are 2 different things

Reply:
👍